### DISCUSSION TITLE
How to get the three input data in the example program（WBPR.srn5）？

### DISCUSSION THREAD

**Discussion by zhifeiji-han** [SCIInstitute/SCIRun#2313] [General]
In the WBPR.srn5 example program, how to obtain  heart.fld, thorax.fld and  bsp.mat. I know it was obtained by using ECGSim software, but I used the software to export without these three files, so I want to ask how to obtain these three files?
I want to get a set of epicardial and body surface potential data to verify the new algorithm, but I don't know how to get it using ecgsim.

**Comment by jab0707** [comment]
What is the form that your geometry data is in after you export from ECG sim? How you get them into SCIRun depends on what format they were exported from ECGsim in.
